db/schema.rb

```diff
@@ -199,20 +199,20 @@
   end
 
   create_table "tasks", id: :bigserial, force: :cascade do |t|
```

Comment:
To reduce noise, please try not to commit schema.rb when the only change is the field order. Unfortunately this sometimes happens when running migrations locally, since schema.rb is dumped from your local DB. 
My habit is: whenever I see schema.rb has changed and I haven't actually changed the DB schema I review the diff. Most of the time I can then do a `git checkout db/schema.rb` to reset the file.